actions/nuxeo-share-button.html

```diff
@@ -45,12 +57,16 @@
     </dom-if>
 
     <nuxeo-dialog id="dialog" with-backdrop>
-      <h2>[[i18n('shareButton.dialog.heading')]]</h2>
-      <paper-input id="permalink" label="[[i18n('shareButton.copy.label', document.properties.dc:title)]]"
+      <div class="heading">
+        <h2>[[i18n('shareButton.dialog.heading')]]</h2>
+        <span id="copyLink" on-click="_copyPermalink">[[i18n('shareButton.operation.copy')]]</span>
+      </div>
+      <nuxeo-input id="permalink"
                    value="[[_buildPermalink(document)]]"
+                   on-click="_copyPermalink"
                    autofocus
-                   readonly>
-      </paper-input>
+                   readonly no-label-float>
```

Comment:
no need to add this `no-label-float` since a) `nuxeo-input` does not have such property (it belongs to `paper-input`); b) `nuxeo-input` already wrap a `paper-input` and therefore set this property by default